dev.skynomads.Seabird.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: dev.skynomads.Seabird
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: seabird
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+
+modules:
+  - name: seabird
+    buildsystem: simple
+    build-commands:
+      - |
+        install -Dm00755 -t /app/bin linux_amd64/seabird
+        install -Dm00644 seabird.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+        install -Dm00644 -t /app/share/applications $FLATPAK_ID.desktop
+        install -Dm00644 -t /app/share/metainfo $FLATPAK_ID.appdata.xml
+    sources:
+      - type: file
+        path: dev.skynomads.Seabird.desktop
+      - type: file
+        path: dev.skynomads.Seabird.appdata.xml
+      - type: file
+        filename: seabird.svg
+        url: https://raw.githubusercontent.com/getseabird/seabird/d1c2366c5e6d5f12f070e446ceb99511f56ab7e8/icon/seabird.svg
+        sha256: 6593267d7c60a8a84c350250854fc3f2929c17af612a3b4e44d958b5301cd4fe
+      - type: archive
+        strip-components: 0
+        dest: linux_amd64
+        url: https://github.com/getseabird/seabird/releases/download/v0.0.19/seabird_linux_amd64.tar.gz
+        sha256: 79fdbc6a992e9a3cacb8526b073f03d8fe084c38b141a9000eb15d27f986401f
```

Comment:
Why not build from source?

Reply:
As written above:

> there's currently no working solution for Go dependencies in builder-tools.

Network access during builds isn't permitted, so this is the only way right now

Reply:
Youn mean tools like this https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get


There is a certain number of go package built on Flathub.

Reply:
They don't work with go modules I don't think. This has been [reported](https://github.com/flatpak/flatpak-builder-tools/issues/285) is a [PR](https://github.com/flatpak/flatpak-builder-tools/pull/369) for this but it's still WIP. Or do you have any examples of go packages using modules? Perhaps I did something wrong

Reply:
so you ask me to know about go. 

I don't.

I have no idea how a go module is different than the other stuff.

Maybe you can help finish the tools.

Reply:
No worries, go redesigned how dependencies are installed a few years ago and `go get` is the legacy tool that's incompatible with the new module system. So, unfortunately, I don't think it's possible to build from source right now. I might be able to help finish the new tool in the future, but there's also an app I'm working on at the moment. :wink:

Reply:
and this new system doesn't support vendoring packages?

Reply:
Oh it does support vendoring, but I don't want to put all dependencies in git. I guess I could generate a source + deps tar.gz on release and use that for the flatpak build. But repackaging the binary should be fine for now, no? It can always be improved later